src/main/java/uk/gov/justice/digital/delius/transformers/AppointmentPatchRequestTransformer.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonpatch.JsonPatch;
+import com.github.fge.jsonpatch.JsonPatchOperation;
+import com.github.fge.jsonpatch.ReplaceOperation;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.justice.digital.delius.config.DeliusIntegrationContextConfig.IntegrationContext;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.StreamSupport;
+
+import static com.fasterxml.jackson.databind.node.TextNode.valueOf;
+import static com.github.fge.jackson.jsonpointer.JsonPointer.of;
+import static java.lang.String.format;
+import static java.util.Optional.ofNullable;
+import static java.util.stream.Collectors.toList;
+
+@Component
+@AllArgsConstructor
+public class AppointmentPatchRequestTransformer {
```

Comment:
See note above - but feels like all this code is required since we went down the PATCH route rather than PUT.

Again happy to see how this plays out, but this amount a code suggests it is overkill

Reply:
Covered by previous comment